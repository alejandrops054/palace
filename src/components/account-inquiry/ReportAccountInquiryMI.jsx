import React from "react";
import {
  Row,
  Col,
  Input,
  Label,
  InputGroup,
  InputGroupText,
  InputGroupAddon,
} from "reactstrap";
import InfoMembersNoContainer from "../info-members/InfoMembersNoContainer";
function ReportAccountInquiryMI() {
  return (
    <>
      <Row>
        <Col
          lg={{ size: 4, offset: 4 }}
          md={{ size: 4, offset: 4 }}
          sm={{ size: 8, offset: 2 }}
        >
          <InputGroup>
            <Label>Application: </Label>
            <Input type="text"></Input>
            <InputGroupAddon addonType="append">
              <InputGroupText>...</InputGroupText>
            </InputGroupAddon>
          </InputGroup>
        </Col>
      </Row>
      <Row>
        <Col
          lg={{ size: 4, offset: 5 }}
          md={{ size: 4, offset: 5 }}
          sm={{ size: 3, offset: 5 }}
        >
          <InfoMembersNoContainer title="Company" text="Company" />
          <InfoMembersNoContainer title="Member Name" text="Name" />
        </Col>
      </Row>
    </>
  );
}

export default ReportAccountInquiryMI;
