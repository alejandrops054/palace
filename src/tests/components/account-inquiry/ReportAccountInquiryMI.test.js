/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react'
import {shallow} from 'enzyme'

import ReportAccountInquiryMI from '../../../components/account-inquiry/ReportAccountInquiryMI'

describe('Pruebas del componente ReportAccountInquiryMI ', () => {
    test('Creacion de ReportAccountInquiryMI', () => {
        
        const wrapper = shallow(<ReportAccountInquiryMI/>)

        expect(wrapper).toMatchSnapshot();
    })
    
})

