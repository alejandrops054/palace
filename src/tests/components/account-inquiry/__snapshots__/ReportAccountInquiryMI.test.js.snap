// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente ReportAccountInquiryMI  Creacion de ReportAccountInquiryMI 1`] = `
<Fragment>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      lg={
        Object {
          "offset": 4,
          "size": 4,
        }
      }
      md={
        Object {
          "offset": 4,
          "size": 4,
        }
      }
      sm={
        Object {
          "offset": 2,
          "size": 8,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <InputGroup
        tag="div"
      >
        <Label
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          Application: 
        </Label>
        <Input
          type="text"
        />
        <InputGroupAddon
          addonType="append"
          tag="div"
        >
          <InputGroupText
            tag="span"
          >
            ...
          </InputGroupText>
        </InputGroupAddon>
      </InputGroup>
    </Col>
  </Row>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Col
      lg={
        Object {
          "offset": 5,
          "size": 4,
        }
      }
      md={
        Object {
          "offset": 5,
          "size": 4,
        }
      }
      sm={
        Object {
          "offset": 5,
          "size": 3,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <InfoMembersNoContainer
        text="Company"
        title="Company"
      />
      <InfoMembersNoContainer
        text="Name"
        title="Member Name"
      />
    </Col>
  </Row>
</Fragment>
`;
